### PR TITLE
 initial upload branding_console.adoc

### PR DESCRIPTION
**Pull request by Jooho**
This is operationalization/branding_console playbook. Now, this doc has only manual configuration but I will also update ansible script url when it is ready.

Trello Card
https://trello.com/c/vDRWIM81/158-2-branding-console-login-page-web-console-template-2

### CODE REVIEW

File: playbooks/operationalizing/branding_console.adoc [hunk 1]
```diff
@@ -0,0 +1,287 @@
+= Branding console
```
Comment (etsauer) [inline playbooks/operationalizing/branding_console.adoc]: let's change the title to "Branding the OpenShift Web Console"

**Comment by etsauer** [comment]
@JaredBurck would you be able to help review this?

File: playbooks/operationalizing/branding_console.adoc (hunk 1 above)
Comment (Jooho) [inline playbooks/operationalizing/branding_console.adoc]: Done

File: playbooks/operationalizing/branding_console.adoc [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+...
+assetConfig:
+  extensions:
+    - name: images
+      sourceDirectory:  
+          /etc/origin/master/stylesheet/images
+      extensionDevelopment: true
+      extensionStylesheets:
+    - /etc/origin/master/stylesheet/css/extension.css
+  logoutURL: ""
```
Comment (sabre1041) [inline playbooks/operationalizing/branding_console.adoc]: I went through and validated setting up a custom template logo and there is minor spacing issues above. The following works:

```
assetConfig:
  extensions:
  - name: images 
    sourceDirectory: /etc/origin/master/stylesheet/images
  extensionsDevelopment: true
  extensionStylesheets:
    - /etc/origin/master/stylesheet/css/extension.css
```

**Comment by Jooho** [comment]
Thanks @sabre1041 . I update the part.

File: playbooks/operationalizing/branding_console.adoc (hunk 2 above)
Comment (Jooho) [inline playbooks/operationalizing/branding_console.adoc]: updated it

**Comment by sabre1041** [comment]
@Jooho Content is ready. However formatting for jekyll has a small omission. Can you add the following to the top of your document so it can get picked up properly?

```
---
---
```

In addition, can you create a link called "Branding the OpenShift Web Console" on the operationalizing index page?

**Comment by Jooho** [comment]
@sabre1041 Please review it ^^

**Comment by sabre1041** [comment]
looks good! Thanks for the great work!

**Pull request merged**